src/Listeners/Healthcheck/Base.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Rapidez\Core\Listeners\Healthcheck;
+
+use Illuminate\Support\Facades\Event;
+
+class Base
+{
+    public function handle()
+    {
+        return [
+            'healthy'  => true,
+            'messages' => [
+                // [
+                //     'type'  => 'error',
+                //     'value' => ''.
+                // ],
+                // [
+                //     'type'  => 'warn',
+                //     'value' => ''.
+                // ],
+                // [
+                //     'type'  => 'info',
+                //     'value' => ''.
+                // ],
```

Comment:
This is just here as an example?

Reply:
Exactly, it is just an example of the expected output. As the handle function of Base is made to be overwritten